tools/lib/subcmd/subcmd-util.h

```diff
@@ -47,6 +47,7 @@ static NORETURN inline void die(const char *err, ...)
 		} \
 	} while(0)
 
+#pragma GCC diagnostic ignored "-Wuse-after-free"
```

Comment:
This should be removed.

Reply:
Yes, it's garbage. Alexander already mentioned this.